test/models/report_test.rb

```diff
@@ -1,7 +1,26 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "report saved" do
```

Comment:
a problem with a test like this, if we add a new validation to the report model this test will break, but that doesn't mean that the app break.